cinema/tests/test_movie_api.py

```diff
@@ -83,9 +83,7 @@ def test_get_movie(self):
         self.assertEqual(response.data["genres"][1]["name"], "Comedy")
         self.assertEqual(response.data["actors"][0]["first_name"], "Kate")
         self.assertEqual(response.data["actors"][0]["last_name"], "Winslet")
-        self.assertEqual(
-            response.data["actors"][0]["full_name"], "Kate Winslet"
-        )
+        self.assertEqual(response.data["actors"][0]["full_name"], "Kate Winslet")
```

Comment:
The 'full_name' field is being accessed in the `test_get_movie` method. Ensure that the serializer used in the API includes this field, or adjust the test to match the actual response structure.